js/bittrex.js

```diff
@@ -136,6 +136,19 @@ module.exports = class bittrex extends Exchange {
                     },
                 },
             },
+            'exceptions': {
+                'APISIGN_NOT_PROVIDED': AuthenticationError,
+                'INVALID_SIGNATURE': AuthenticationError,
+                'INVALID_CURRENCY': ExchangeError,
+                'INVALID_PERMISSION': AuthenticationError,
+                'INSUFFICIENT_FUNDS': InsufficientFunds,
+                'QUANTITY_NOT_PROVIDED': InvalidOrder,
+                'MIN_TRADE_REQUIREMENT_NOT_MET': InvalidOrder,
+                'ORDER_NOT_OPEN': InvalidOrder,
+                'UUID_INVALID': OrderNotFound,
+                'RATE_NOT_PROVIDED': InvalidOrder, // createLimitBuyOrder ('ETH/BTC', 1, 0)
+                'Insufficient funds!': InsufficientFunds, // bluetrade
```

Comment:
I'd move this line to bluetrade.js upon merging, if you don't mind )

Reply:
Oops... completely overlooked it.
Actually, bleutrade error messages seem to be quite different anyway, added some of them.

Reply:
Thx for fixing it, I didn't mean to force it on you, was going to merge %) anyways, thanks for the added messages as well.

Reply:
Your questions just forced me to think that I completely forgot to test bleutrade, so I had to revert to it anyway.